ontology/uco/tool/tool.ttl

```diff
@@ -54,139 +60,54 @@ tool:BuildFacet
 tool:BuildInformationType
 	a
 		owl:Class ,
+		owl:DeprecatedClass ,
```

Comment:
@sbarnum : With this deprecation, `tool:buildInformation` is constrained to use a deprecated class in `tool:buildFacet`.  Should `tool:BuildFacet` also be deprecated and/or migrated to `observable:`?

A potential conflict - the class `observable:Build` added in this proposal is a subclass of `ObservableAction`.